app/views/vendors/show_vendor.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @vendor.name %></h1>
+
+<p>Number of Employees: <%= @vendor.no_of_employees %></p>
+
+<p>This Vendor's Products:</p>
+<ul>
+  <% @all_products.each do |product| %>
+    <% if product.vendor_id == @vendor.id %>
```

Comment:
I made a comment in the controller, but here you can just get the specific products of the vendor in the controller instead of doing that check here. That would help get some logic out of the view and makes for better code!